### DISCUSSION TITLE
Nvim-telescope and its dependency plenary.nvim

### DISCUSSION THREAD

**Discussion by refaelsh** [nix-community/nixvim#2095] [Q&A] [answered]
In Nvim-telescope's README is states that:
```
[nvim-lua/plenary.nvim](https://github.com/nvim-lua/plenary.nvim) is required.
```
How does Nixvim's `telescope` option handles this?
`plenary.nvim` is not part of Nixim, it is part of Nixpkgs, so I cant just simply do `programs.nixvim.plugins.plenary.enable = true;`.

**Comment by traxys** [comment, marked as answer]
It is automatically handled by nixpkgs: https://github.com/NixOS/nixpkgs/blob/a24a4fd31612a05c5b2d0e1d4ab928cbebcb5f42/pkgs/applications/editors/vim/plugins/overrides.nix#L1712-L1714

**Reply by refaelsh** [reply]
Understood. Thank you.
